### PR TITLE
Update decsription in TCP glossary

### PR DESCRIPTION
**Pull request by H0dah**

### CODE REVIEW

**Comment by H0dah** [comment]
Concurrency Control don't mean (initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.)

**Comment by H0dah** [comment]
Concurrency Control don't mean (initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.)

**Comment by wbamberg** [comment]
I wonder if this was supposed to be "congestion control", which would be a closer match to the description, and seems like more of a core feature of TCP...

**Comment by teoli2003** [comment]
I think "congestion control" was what was intended.

File: files/en-us/glossary/tcp/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ page-type: glossary-definition
 
 **TCP (Transmission Control Protocol)** is an important network {{Glossary("protocol")}} that lets two hosts connect and exchange data streams. TCP guarantees the delivery of data and packets in the same order as they were sent. Vint Cerf and Bob Kahn, who were DARPA scientists at the time, designed TCP in the 1970s.
 
-TCP's role is to ensure the packets are reliably delivered, error free. TCP has [concurrency control](https://en.wikipedia.org/wiki/Concurrency_control), which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.
+TCP's role is to ensure the packets are reliably delivered, error free. TCP has [concurrency control](https://en.wikipedia.org/wiki/Concurrency_control), which means the correct results for concurrent operations are generated, while getting those results as quickly as possible.
```
Comment (teoli2003) [inline files/en-us/glossary/tcp/index.md:9]:
```suggestion
TCP's role is to ensure the packets are reliably delivered, error free. TCP implements [congestion control](https://en.wikipedia.org/wiki/TCP_congestion_control), which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.
```

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

This PR looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! ✌️ 

If you're looking for something else to help out with, we have a [contributor board](https://github.com/orgs/mdn/projects/25), and another useful place to look is the list of issues labeled [`good first issue`](https://github.com/mdn/content/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

**Comment by H0dah** [comment]
You're welcome Jean!

I'll check the links you provide, thanks for sharing!

have a good day,

**Pull request merged**